eng/common/scripts/Delete-RemoteBranches.ps1

```diff
@@ -1,46 +1,69 @@
 param(
   [Parameter(Mandatory = $true)]
-  $RepoOwner,
+  # Please use the RepoOwner/RepoName format: e.g. Azure/azure-sdk-for-java
+  $Repo,
```

Comment:
Do we use this script anywhere else; it is under eng/common? If so, changing these parameters will break them.